elasticlib/Elastic.java

```diff
@@ -33,10 +33,29 @@ public static class ElasticNotification {
     @JsonProperty("description")
     private String description;
 
-    public ElasticNotification(NotificationLevel level, String title, String description) {
+    @JsonProperty private int displayTime;
+
+    @JsonProperty private double width;
+
+    @JsonProperty private Double height;
```

Comment:
The `JsonProperty` should specify the key (like how it's with description) so that way it's not tied to the variable name

Also for height, there isn't really a need for it to be a `Double`